### PR TITLE
Feature/entities

### PR DESCRIPTION
**Pull request by teasmade**
- Add TypeORM entities to correspond to MPD
- Add minimal fetch of roles with a user relation in client to test connections
- Refactor entities to use a centralised index file for cleaner imports

### CODE REVIEW

**Review by Adri-bis — DISMISSED** [review]

**Review by Adri-bis — DISMISSED** [review]

File: core_api/src/modules/patient/patient.entity.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  @Field(() => String)
+  @Column({ nullable: false, unique: true, type: 'varchar', length: 100 })
+  ssn: string;
+
+  @Field(() => String)
+  @Column({ nullable: false, type: 'varchar', length: 100 })
+  town: string;
+
+  @Field(() => String)
+  @Column({ nullable: false, type: 'varchar', length: 100 })
```
Comment (julien44830) [inline core_api/src/modules/patient/patient.entity.ts]: `  @Column({ nullable: false, type: 'varchar', length: 100 })`

Peut-être un peut long pour un code postal

File: core_api/src/modules/user/user.entity.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  @Field(() => String)
+  @Column({ nullable: false, type: 'varchar', length: 50 })
+  lastname: string;
+
+  @Field(() => String)
+  @Column({ nullable: false, unique: true, type: 'varchar', length: 100 })
+  email: string;
+
+  @Field(() => String)
+  @Column({ nullable: false, type: 'varchar', length: 100 })
```
Comment (julien44830) [inline core_api/src/modules/user/user.entity.ts]: `@Column({ nullable: false, type: 'varchar', length: 100 })`

Anticiper un hash de MDP plus long que 100 caractère

**Review by julien44830 — CHANGES_REQUESTED** [review]

File: core_api/src/modules/patient/patient.entity.ts (hunk 1 above)
Comment (teasmade) [reply core_api/src/modules/patient/patient.entity.ts]: done

File: core_api/src/modules/user/user.entity.ts (hunk 2 above)
Comment (teasmade) [reply core_api/src/modules/user/user.entity.ts]: done

**Review by julien44830 — APPROVED** [review]

**Review by Adri-bis — APPROVED** [review]

**Review by Acetoshi — APPROVED** [review]
OK

**Pull request merged**